core/src/main/scala/pink/cozydev/protosearch/Scorer.scala

```diff
@@ -17,12 +17,15 @@
 package pink.cozydev.protosearch
 
 import cats.data.NonEmptyList
-import cats.syntax.all._
+import cats.syntax.all.*
```

Comment:
Huh, I didn't realize the project used a mix of `_` and `*` imports already, oops.
I wonder if there's a scalafmt setting to rewrite them one way or the other.
Created a new issue for trying to standardize and enforce consistency: https://github.com/cozydev-pink/protosearch/issues/195

(this is fine for now though, no change required)

Reply:
Interesting. I haven't really touched scalafmt before so I wasn't sure if it even cleaned up imports in the first place.